### ISSUE TITLE
FormBuilderChipsInput : 'inputType != null': is not true (and other errors)

### ISSUE THREAD

**Issue by Sigfrodr** [danvick/flutter_form_builder#152]
The FormBuilderChipsInput does not actually works without adding the following variables:
```
inputType: TextInputType.text,
obscureText: false,
autocorrect: false,
keyboardAppearance: Brightness.dark,
inputAction: TextInputAction.done,
textCapitalization: TextCapitalization.none,
```

**Comment by Laban1** [comment]
Same issue here.  Just downloaded from Git and it does not run without the changes pointed out by the OP.

**Comment by danvick** [comment]
Fixed in v3.6.0.
